src/org/zaproxy/zap/extension/openapi/ExtensionOpenApi.java

```diff
@@ -229,7 +229,7 @@ public void run() {
                     }
                 } catch (Exception e) {
                     if (initViaUi) {
-                        View.getSingleton().showWarningDialog(Constant.messages.getString("openapi.parse.error"));
+                        View.getSingleton().showWarningDialog(Constant.messages.getString("openapi.parse.error", e.getMessage()));
```

Comment:
We should hardcode a limit for the text shown in the dialogue (or change the messages?). Some of the messages include the whole definition being imported (e.g. try import a JSON definition with a missing curly bracket).

Might worth internationalise the message(s) now that they are being shown in the UI.

Reply:
That would not work as is, they are (just) in English (as thrown from `SwaggerConverter`).

That length seems to be enough. Does it worth mentioning that the remaining is in the log?

Reply:
Ok I'm not sure what we're after on the i18n front then, as SwaggerConverter is just an implementation of Converter from a library we're using and SwaggerException just extends Exception and calls super methods.... We could extract strings and try to maintain the i18n but that just seems like a headache. Or am I missing something?

Reply:
Ok I'll have to look around more, I didn't notice them.

Reply:
Geez if it'd been a snake it woulda bitten me....that's how close it was. Update coming.